benchbuild/projects/gentoo/portage_gen.py

```diff
@@ -55,7 +111,9 @@ def run_not_supported(self, *args, **kwargs): # pylint: disable=W0613
     newclass = type(name, (BaseClass,), {
         "NAME" : NAME,
         "DOMAIN" : DOMAIN,
+        "VERSION" : FuncClass(NAME, DOMAIN),
         "GROUP" : "auto-gentoo",
+        "SRC_FILE" : "automatic",
```

Comment:
It is not needed and I removed it, since the gentoo projects gather their version number without a source file to get the hash from.
Should be gone with the next commit.